website/docs/r/lb_target_group.html.markdown

```diff
@@ -94,6 +94,7 @@ load balancers at this time. See
 http://docs.aws.amazon.com/elasticloadbalancing/latest/APIReference/API_CreateTargetGroup.html
 for a complete reference.
 
+* `enabled` - (Optional) Indicates whether  health checks are enabled. Defaults to true.
```

Comment:
I still think, enabling the health check in case the user adds an `health_check {}` block is a good idea. What is the point in configuring the checks but disabling them?

It defaults to API default in cases where the `health_check {}` block is omitted. Which is, disabled for lambda, enabled for others.

I commented on the docs to point out, that health checks are actual requests. No matter if it is a lambda or a HTTP endpoint that is called.